docs/devel/translations.rst

```diff
@@ -6,54 +6,47 @@ Managing translations
 Adding new translations
 -----------------------
 
-Weblate can add new translations to your translation components when there is a configured
-:guilabel:`Template for new translations` (see :ref:`component`), or when your file
-format doesn't require a template (for most monolingual files it is okay to start
-with blank files).
+New strings can be made available for translation when they appear in the base file,
+called :guilabel:`Template for new translations` (see :ref:`component`).
+If your file format doesn't require such a file, as is the case with most monolingual
+translation flows, you can start with blank files).
 
-Weblate can be configured to automatically add a translation when requested by a
-user or to send notification to project admins for approval and manual
-processing. This can be done using :guilabel:`Start new translation` in
-:ref:`component`. The project admins can still start translation within Weblate
-even if the contact form is shown for regular users.
+New languages can be added right away when requested by a user in Weblate, or a
+notification will be sent to project admins for approval and manual addition.
+This can be done using :guilabel:`Start new translation` in :ref:`component`.
 
-Alternatively you can add the files manually to the VCS. Weblate will
-automatically detect new languages which are added to the VCS repository and
-will make them available for translation. This makes adding new translations
-incredibly easy:
+.. note::
 
-1. Add the translation file to VCS.
-2. Let Weblate update the repository (usually set up automatically, see
-   :ref:`update-vcs`).
+    Project admins can always start translation within Weblate directly.
 
+Language files added manually to the VCS are added when Weblate updates
+the repository if that is set up, see :ref:`update-vcs`).
```

Comment:
When automatic addition or updating the repository is set up?

Reply:
with every repo update. It depends when based on settings. Thanks for the heads up!